net-misc/ptpd/ptpd-2.3.0.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit autotools eutils flag-o-matic systemd
+
+DESCRIPTION="Precision Time Protocol daemon"
+HOMEPAGE="http://ptpd.sf.net"
+
+if [[ ${PV} == "9999" ]]; then
+	ESVN_REPO_URI="http://svn.code.sf.net/p/ptpd/code/trunk"
+	inherit subversion
+	SRC_URI=""
+	KEYWORDS=""
+else
+	if [[ ${PV} == *_rc* ]]; then
+		MY_PV=${PV/_rc*/}
+		MY_P=${P/_rc/-rc}
```

Comment:
I don't think doing those substitutions on non-rc P/PV would change anything. So why not remove the conditional and just do them unconditionally?